app/pubhub/v1/user/loans/[identifier]/route.ts

```diff
@@ -0,0 +1,34 @@
+import { NextRequest, NextResponse } from "next/server"
+
+import { withAuth } from "@/app/pubhub/(lib)/helper"
+import { TUserInfo } from "@/app/pubhub/(lib)/types"
+import { getLastValueFromUrl, transformTimeToUtcString } from "@/app/pubhub/helper"
+
+import { createLoanRequest } from "./(lib)/requests"
+import { createLoanSchema } from "./(lib)/schemas"
+
+async function postLibraryLoan(request: NextRequest, context: { userInfo: TUserInfo }) {
```

Comment:
I think we should call the function `createLoan` or `createdigitalLoan`. The `POST` export later on shows that we are dealing with a POST request so we can instead focusing on calling our function something human readable :)